app/assets/javascripts/pretty-text/addon/pretty-text.js

```diff
@@ -33,22 +33,14 @@ export function buildOptions(state) {
     emojiUnicodeReplacer,
     lookupUploadUrls,
     previewing,
-    linkify,
```

Comment:
Passing this option in has never had any effect since the markdown engine is using the site setting directly.